apps/admin-gui/src/app/shared/side-menu/side-menu-item.service.ts

```diff
@@ -322,6 +289,48 @@ export class SideMenuItemService {
     };
   }
 
+  private getResourceLinks(baseUrl: string, regexStart: string, resource: Resource) {
+    const links: EntityMenuLink[] = [{
+      label: 'MENU_ITEMS.RESOURCE.OVERVIEW',
+      url: [baseUrl],
+      activatedRegex: `${regexStart}/\\d+/resources/\\d+$`
+    }];
+    if(this.guiAuthResolver.isAuthorized('getAssignedGroups_Resource_policy', [resource])){
+      links.push({
+        label: 'MENU_ITEMS.RESOURCE.ASSIGNED_GROUPS',
+        url: [baseUrl, 'groups'],
+        activatedRegex: `${regexStart}/\\d+/resources/\\d+/groups$`
+      })
+    }
+    if(this.guiAuthResolver.isPerunAdmin()){
+      links.push({
+        label: 'MENU_ITEMS.RESOURCE.ASSIGNED_SERVICES',
+        url: [baseUrl, 'services'],
+        activatedRegex: `${regexStart}/\\d+/resources/\\d+/services$`
+      })
```

Comment:
We made some misscomunication, to get into the assigned services and see them/add/remove them should be done according to the policies.